content/F2016/F16-Office-Hours.md

```diff
@@ -0,0 +1,33 @@
+Title: Office Hours for Fall 2016
+Date: 2016-10-06 19:00
+Category: Blog
+Tags: Office Hours
+Author: Rolina Wu
+Summary: Announcing our Office Hours for Fall 2016
+
+Come visit us during our Office hours or just swing by to say hi! We have various technical interview resources and publications available for you to read during Office Hours. There's always an officer that will be available in the office if you have any questions or want to meet some new faces. :)
+You can also read more about what our office provides on our [contact us](https://wics-site-pr-267.herokuapp.com/contact-us/) page.
+
+Monday:
+10am - 1 pm
+ 2pm - 4 pm
+
+Tuesday:
+ 2pm - 4 pm
+
+Wednesday:
+10am - 11am
+12pm - 1 pm
+ 2pm - 4 pm
+ 
+Thursday:
+ 2pm - 4 pm
+
+Friday:
+10am - 11am
+ 2pm - 4 pm
+
+No office hours will be hold on Saturday and Sunday.
```

Comment:
Hey quick note:
I think it should be "No office hours will be held on Saturday and Sunday"
Thanks!